plugins/explore/src/components/DefaultExplorePage/DefaultExplorePage.tsx

```diff
@@ -32,7 +32,10 @@ export const DefaultExplorePage = () => {
       subtitle="Discover solutions available in your ecosystem"
     >
       <ExploreLayout.Route path="domains" title="Domains">
-        <DomainExplorerContent />
+        <EntityExplorerContent kind="domain" />
+      </ExploreLayout.Route>
+      <ExploreLayout.Route path="systems" title="Systems">
+        <EntityExplorerContent kind="system" />
```

Comment:
I don't think we should add systems by default tbh. It's going to be a very large amount for many orgs, more than you'd wish to browse on this page